pyro/infer/autoguide/guides.py

```diff
@@ -272,22 +272,21 @@ def my_init_fn(site):
     :param callable init_loc_fn: A per-site initialization function.
         See :ref:`autoguide-initialization` section for available functions.
     """
-    def __init__(self, model, prefix="auto", init_loc_fn=init_to_median):
+    def __init__(self, model, init_loc_fn=init_to_median):
         self.init_loc_fn = init_loc_fn
         model = InitMessenger(self._init_loc_fn)(model)
-        super().__init__(model, prefix=prefix)
+        super().__init__(model)
 
     def _setup_prototype(self, *args, **kwargs):
         super()._setup_prototype(*args, **kwargs)
 
         # Initialize guide params
         for name, site in self.prototype_trace.iter_stochastic_nodes():
-            param_name = "{}_{}".format(self.prefix, name)
             value = PyroParam(site["value"].detach(), constraint=site["fn"].support)
-            setattr(self, param_name, value)
+            setattr(self, name, value)
 
     def _init_loc_fn(self, site):
-        name = "{}_{}".format(self.prefix, site["name"])
+        name = "{}.{}".format(self._pyro_name, site["name"])
```

Comment:
Is it safe to remove this param store lookup code now? I'm guessing this was to allow users to set control the initialization?

Reply:
:+1: Yes I believe so. Good catch.